webapp-django/crashstats/symbols/views.py

```diff
@@ -122,6 +139,7 @@ def api_upload(request):
 
 @require_POST
 @csrf_exempt
+@api_login_required
 @permission_required('crashstats.upload_symbols')
```

Comment:
The `@permission_required` basically wraps `login_required` but if the check got down to that it would not be possible to control the error if the Auth-Token failed.